contracts/VeRBNRewards.sol

```diff
@@ -0,0 +1,247 @@
+        gov = _gov;
+    }
+
+    modifier _updateReward(address account) {
+        rewardPerTokenStored = rewardPerToken();
+        lastUpdateTime = lastTimeRewardApplicable();
+
+        if (account != address(0)) {
+            rewards[account] = _earnedReward(account);
+            userRewardPerTokenPaid[account] = rewardPerTokenStored;
+        }
+        _;
+    }
+
+    /**
+     *  @return timestamp until rewards are distributed
+     */
+    function lastTimeRewardApplicable() public view returns (uint256) {
+        return Math.min(block.timestamp, periodFinish);
+    }
+
+    /** @notice reward per token deposited
+     *  @dev gives the total amount of rewards distributed since inception of the pool per vault token
+     *  @return rewardPerToken
+     */
+    function rewardPerToken() public view returns (uint256) {
+        uint256 supply = veToken.totalSupply();
+        if (supply == 0) {
+            return rewardPerTokenStored;
+        }
+        return
+            rewardPerTokenStored +
+            (((lastTimeRewardApplicable() - lastUpdateTime) *
+                rewardRate *
+                1e18) / supply);
+    }
+
+    function _earnedReward(address account) internal view returns (uint256) {
+        return
+            (veToken.balanceOf(account) *
+                (rewardPerToken() - userRewardPerTokenPaid[account])) /
+            1e18 +
+            rewards[account];
+    }
+
+    /** @notice earning for an account
+     *  @return amount of tokens earned
+     */
+    function earned(address account) external view returns (uint256) {
+        return _earnedReward(account);
+    }
+
+    /** @notice use to update rewards on veRBN balance changes.
+        @dev called by veRBN
+     *  @return true
+     */
+    function updateReward(address _account)
+        external
+        _updateReward(_account)
+        returns (bool)
+    {
+        require(msg.sender == address(veToken), "!authorized");
+
+        return true;
+    }
+
+    /**
+     * @notice
+     *  Get rewards for an account
+     * @dev rewards are transfer to _account
+     * @param account to claim rewards for
+     * @return true
+     */
+    function getRewardFor(address account) external returns (bool) {
+        _getReward(account, false);
+        return true;
+    }
+
+    /**
+     * @notice
+     *  Get rewards
+     * @param _lock should it lock rewards into veRBN
+     * @return true
+     */
+    function getReward(bool _lock) external returns (bool) {
+        _getReward(msg.sender, _lock);
+        return true;
+    }
+
+    /**
+     * @notice
+     *  Get rewards
+     * @return true
+     */
+    function getReward() external returns (bool) {
+        _getReward(msg.sender, false);
+        return true;
+    }
+
+    function _getReward(address _account, bool _lock)
+        internal
+        _updateReward(_account)
+    {
+        uint256 reward = rewards[_account];
+        rewards[_account] = 0;
+        if (reward == 0) return;
```

Comment:
I'd return 1 line earlier before `rewards[_account] = 0;`.  Makes the code a bit easier to read and doesn't cause any issues